templates/default/service.erb

```diff
@@ -32,6 +32,11 @@ STOP_SERVER_CMD="$BIN_DIR/$SCRIPT-stop 2> /var/log/confluent/$NAME-service.out"
 export <%= k %>="<%= v %>"
 <% end -%>
 
+##For Kafka Monitoring
+KAFKA_PIDFILE=/var/run/$NAME.pid
+mkdir -p $(dirname $KAFKA_PIDFILE)
+chown $USER $(dirname $KAFKA_PIDFILE) || true
```

Comment:
I don't think we want the confluent user owning `/var/run`. I believe its more typical to create a directory in `/var/run` which is owned by the service user. That creation logic should be moved into the `_install` recipe